packages/babel-plugin-transform-es2015-computed-properties/src/index.js

```diff
@@ -1,4 +1,9 @@
-export default function({ types: t, template }) {
+export default function({ types: t, template }, options) {
+  const { loose: isLoose } = options;
+
+  let callback = spec;
```

Comment:
personally I would rename `callback` to smth like `pushComputedProps`, and accordingly:
- `spec` -> `pushComputedPropsSpec`
- `loose` -> `pushComputedPropsLoose`

to give them all more meaning